lib/app.js

```diff
@@ -0,0 +1,30 @@
+(function (factory) {
```

Comment:
This is the old app.js file moved to the lib folder. It has been wrapped in an IIFE so that it can work with AMD, CommonJS (Node) and the browser (globals). Also we are calling ko.applyBindings with `new App()` which is a new "view model" responsible for holding the calculator, and offering methods to add/remove calculators.